redash/models/__init__.py

```diff
@@ -315,7 +315,7 @@ class QueryResult(db.Model, BelongsToOrgMixin):
     data_source = db.relationship(DataSource, backref=backref("query_results"))
     query_hash = Column(db.String(32), index=True)
     query_text = Column("query", db.Text)
-    data = Column(MutableDict.as_mutable(JSONB), nullable=True)
```

Comment:
This was

```
_data = Column("data", db.Text)
```

Reply:
thought we can use the benefits of sqlalchemy JSON class and have Text type in a db, but unfortunately it converts column automatically
converted to Text column